requirements/base.txt

```diff
@@ -4,59 +4,64 @@
 #
 #    make upgrade
 #
-amqp==2.6.1               # via kombu
-arrow==0.15.8             # via -r requirements/base.in
-attrs==19.3.0             # via glom
+amqp==5.0.5               # via kombu
+arrow==0.17.0             # via -r requirements/base.in
+attrs==20.3.0             # via glom
 billiard==3.6.3.0         # via celery
 blinker==1.4              # via -r requirements/base.in, sentry-sdk
-boltons==20.2.0           # via face, glom
-cachetools==4.1.1         # via -r requirements/base.in
-celery==4.4.7             # via -r requirements/base.in
-certifi==2020.6.20        # via requests, sentry-sdk
-cffi==1.14.1              # via cryptography
-chardet==3.0.4            # via requests
-click==7.1.2              # via -r requirements/base.in, flask, rq
-cryptography==3.0         # via -r requirements/base.in, jwcrypto, oauthlib
+boltons==20.2.1           # via face, glom
+cachetools==4.2.1         # via -r requirements/base.in
+celery==5.0.5             # via -r requirements/base.in
```

Comment:
Is celery 5 a no-touch upgrade?  I've seen scary conversations about Celery.

Reply:
We have updated almost all packages to `celery` 5.0 only services are left and services are blocked due to the `flower` support issue in `celery` 5.0

Reply:
I don't know what the flower support issue is.  Does it affect this repo?

Reply:
`flower` is a GUI based celery tasks monitoring tool, which we are using in IDAs to monitor celery tasks